README.md

```diff
@@ -0,0 +1,12 @@
+# Voltz OSS
+
+An open-sourced Voltz remake.
```

Comment:
Remember, the goal of this project is to be a logical continuation of Voltz, not a remake. (despite actually being a remake)